common/js/src/i18n-unittest.js

```diff
@@ -0,0 +1,103 @@
+/** @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+goog.require('GoogleSmartCard.I18n');
+goog.require('goog.dom');
+goog.require('goog.testing');
+goog.require('goog.testing.jsunit');
+
+goog.setTestOnly();
+
+goog.scope(function() {
+
+const GSC = GoogleSmartCard;
+
+const propertyReplacer = new goog.testing.PropertyReplacer;
+/** @type {Element?} */
+let pNode;
+/** @type {Element?} */
+let p2Node;
```

Comment:
nit: I would rename this to pNode2 as it is another node of type p - p2Node could be misunderstood that it is a node <p2>. Optionally, pNode could also be renamed to pNode1 for consistency.